python/cudf/cudf/tests/test_reshape.py

```diff
@@ -621,3 +621,32 @@ def test_crosstab_simple():
     expected = pd.crosstab(a, [b, c], rownames=["a"], colnames=["b", "c"])
     actual = cudf.crosstab(a, [b, c], rownames=["a"], colnames=["b", "c"])
     assert_eq(expected, actual, check_dtype=False)
+
+
+@pytest.mark.parametrize(
+    "values", ["z", "z123", ["z123"], ["z", "z123", "123z"]]
+)
+def test_pivot_values(values):
```

Comment:
I'd prefer to see this test placed next to the other `pivot` tests instead of at the bottom, below the `crosstab` tests.

Reply:
Done.